### PR TITLE
Changes to build and install sfc_climo_gen

### PR DESCRIPTION
**Pull request by LarissaReames-NOAA**

### CODE REVIEW

File: sorc/regional_build.cfg [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
+ Building forecast (forecast) .......................... no
+ Building gsi (gsi) .................................... no
+ Building post (post) .................................. no
  Building utils (utils) ................................ yes
  Building chgres (chgres) .............................. yes
  Building chgres_cube (chgres_cube) .................... yes
+ Building sfc_climo_gen (sfc_climo_gen) ................ yes
  Building orog (orog) .................................. yes
- Building fre-nctools (nctools) ........................ yes
+ Building fre-nctools (nctools) ........................ no
```
Comment (ghost) [inline sorc/regional_build.cfg]: Do we want forecast, post, and fre-nctools to default to "no" for the build script?

File: sorc/build_utils.sh [hunk 2]
```diff
@@ -54,6 +54,14 @@ echo " .... Building fre-nctools .... "
 ./build_fre-nctools.sh > $logs_dir/build_fre-nctools.log 2>&1
 }
 
+#------------------------------------
+# build sfc_climo_gen
+#------------------------------------
+$Build_nctools && {
```
Comment (ghost) [inline sorc/build_utils.sh]: Should "$Build_nctools" be in this code block?

File: sorc/regional_build.cfg (hunk 1 above)
Comment (LarissaReames) [reply sorc/regional_build.cfg]: This is fixed in the latest merge from @BinLiu-NOAA 's branch

File: sorc/build_utils.sh (hunk 2 above)
Comment (LarissaReames) [reply sorc/build_utils.sh]: This is fixed in the latest merge from @BinLiu-NOAA 's branch

**Comment by JacobCarley-NOAA** [comment]
Is this change ready for merger?

**Comment by BinLiu-NOAA** [comment]
Yes, please help to merge into the develop branch.

I have tested on theia, cray, and dell, and approved the PR.

Thanks!

**Comment by JacobCarley-NOAA** [comment]
@BinLiu-NOAA Excellent! Thanks!

**Pull request merged**